addons/account_sequence/tests/test_ir_sequence_concurrency.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import psycopg2
+
+from odoo import SUPERUSER_ID, api, fields
+from odoo.tests import tagged
+from odoo.tools import mute_logger
+from odoo.addons.account.tests.test_sequence_mixin import TestSequenceMixinConcurrency
+
+
+@tagged('post_install', '-at_install', 'test_ir_sequence_concurrency')
+class TestIrSequenceConcurrency(TestSequenceMixinConcurrency):
```

Comment:
Importing and inheriting this class will lead to running the tests 3 times
* first time when running the tests of `account`, where the class is defined
* second time when importing it in this file, since the test suite will detect the name `TestSequenceMixinConcurrency` in a test file and run it
* third time when running all the tests of `TestIrSequenceConcurrency` since it inherited the tests of the parent class

In order to avoid that, split the setup and the tests of `TestSequenceMixinConcurrency` in 2 different classes and inherit only the setup from here